$NetBSD: patch-at,v 1.6 2018/06/19 02:04:04 maya Exp $

Support shared libraries.
Don't capture value of HOME.

No non-local gotos on powerpc32, too

--- configure.orig	2014-09-08 04:56:03.000000000 +0000
+++ configure
@@ -1514,7 +1514,7 @@ Installation directories:
 By default, \`make install' will install all the files in
 \`$ac_default_prefix/bin', \`$ac_default_prefix/lib' etc.  You can specify
 an installation prefix other than \`$ac_default_prefix' using \`--prefix',
-for instance \`--prefix=\$HOME'.
+for instance \`--prefix=\\\$HOME'.
 
 For better control, use the options below.
 
@@ -9318,8 +9318,8 @@ case "$ac_cv_c_compiler_gnu" in yes)
             mercury_cv_asm_labels=no
             mercury_cv_gcc_labels=no
             ;;
-        powerpc64-*)
-            echo "gcc labels do not work on PPC64" 1>&6
+        powerpc*-*)
+            echo "gcc labels do not work on PPC" 1>&6
             mercury_cv_asm_labels=no
             mercury_cv_gcc_labels=no
             ;;
@@ -11481,7 +11481,7 @@ $as_echo "yes" >&6; }
         EXT_FOR_SHARED_LIB=so
         EXT_FOR_LINK_WITH_PIC_OBJECTS=o
         ;;
-    i*86-*-freebsd*)
+    i*86-*-freebsd*|*-netbsd*)
         # From Cornelis van der Laan <nils@ims.uni-stuttgart.de>
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
@@ -13143,7 +13143,7 @@ if test $mercury_cv_have_readline = no; 
 
 else
 	TERMCAP_LIBRARY="$mercury_cv_termcap_lib"
-	READLINE_LIBRARIES="-lreadline $TERMCAP_LIBRARY"
+	READLINE_LIBRARIES="-L${READLINE_PREFIX}/lib ${LINKER_RPATH_FLAG}${READLINE_PREFIX}/lib -lreadline $TERMCAP_LIBRARY"
 fi
 
 
